rasa/core/channels/slack.py

```diff
@@ -399,7 +457,16 @@ async def health(_: Request) -> HTTPResponse:
         @slack_webhook.route("/webhook", methods=["GET", "POST"])
         async def webhook(request: Request) -> HTTPResponse:
             content_type = request.headers.get("content-type")
-            # Slack API sends either a JSON-encoded or a URL-encoded body depending on the content
+
+            if not self.is_request_from_slack_authentic(request):
+                return response.text(
+                    "Message is not properly signed with a valid "
+                    "X-Slack-Signature header",
```

Comment:
should we include a link to the docs?

Reply:
I don't think that is necessary, the user won't see this response as it is returned to slack in the most likely scenario when they are using the wrong signature